web-client/Cargo.toml

```diff
@@ -21,12 +21,18 @@ maintenance = { status = "experimental" }
 crate-type = ["cdylib"]
 
 [dependencies]
+beserial = { path = "../beserial", features = ["derive"] }
```

Comment:
This is actually an internal crate, we regularly try to list them apart (see suggestion below)